requirements_test.txt

```diff
@@ -1,5 +1,6 @@
-coverage==3.7.1
-mock==1.0.1
-nose==1.3.3
-flake8==2.3.0
-objgraph==3.4.0
+coverage
```

Comment:
Is this still needed since pytest-cov also have this dep?

Reply:
removed